extensions/azurecore/src/account-provider/utils/msalCachePlugin.ts

```diff
@@ -64,8 +64,9 @@ export class MsalCachePluginProvider {
 				}
 				else {
 					Logger.error(`MsalCachePlugin: Failed to read from cache file: ${e}`);
-					Logger.verbose(`MsalCachePlugin: Error occurred when trying to read cache file, file contents will be cleared: ${e.message}`);
-					await fsPromises.writeFile(this._msalFilePath, '', { encoding: 'utf8' });
+					Logger.verbose(`MsalCachePlugin: Error occurred when trying to read cache file, file will be deleted: ${e.message}`);
+					await fsPromises.unlink(this._msalFilePath);
+					// await fsPromises.writeFile(this._msalFilePath, '', { encoding: 'utf8' });
```

Comment:
should we remove this commented out code or add some text why it is being retained?

Reply:
Oh yes, I accidentally missed it.